app/views/inventory/manual.snapshot.controller.js

```diff
@@ -39,46 +39,46 @@ function snapshotManualCtrl ($scope, android, fields, $state, enums, resources)
         new fields.Select('device.type', {
```

Comment:
I'm not 100% sure since I haven't practically touched the client. But if we remove line 34:
actions: [
              {
                type: 'VisualTest'
              }
            ]
I would say that the current problem is solved:
Currently, when you make a request, a Visual Test action is sent when it is made from the client. When you make a POST to create a device, it gives an error because it has no appareance or functionality range, even though these are no longer mandatory fields for the form.